montecarlo/vmc/inc/TMCParticleStatus.h

```diff
@@ -48,7 +48,7 @@ struct TMCParticleStatus {
    /// Print all info at once
    void Print() const
    {
-      ::Info("Print", "Status of track");
```

Comment:
This should not be reverted! Please make sure this wasn't reverted in master either!